crypto/rand/rand_win.c

```diff
@@ -34,52 +34,44 @@
 #  define INTEL_DEF_PROV L"Intel Hardware Cryptographic Service Provider"
 # endif
 
-static void readtimer(void);
-
 int RAND_poll(void)
 {
-    MEMORYSTATUS mst;
 # ifndef RAND_WINDOWS_USE_BCRYPT
     HCRYPTPROV hProvider;
 # endif
     DWORD w;
     BYTE buf[64];
```

Comment:
since we use 256 elsewhere.  We should think about changing it to 128.  But I did fix this to use RANDOMNESS_NEEDED from rnd_lcl, so at least it's consistent.